### ISSUE TITLE
Messages are not sent after coming back online

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#21394]
**Preconditions:** User A and B are mutual contacts

**Steps:** 

1. Logout by User A and B
2. Disconnect form internet by both users
3. Login by both users
4. User A sends stack of messages to User B
5. Re connect to internet by both users
6. See if messages are sent and delivered 

**Actual result:** messages are stuck greyed out for sender (User A). New messages also not being sent. The messages start being sending only after relogin

[andr_User_B_receiver_logs.zip](https://github.com/user-attachments/files/17260108/andr_User_B_receiver_logs.zip)
[sender_User_A_ios_logs.zip](https://github.com/user-attachments/files/17260112/sender_User_A_ios_logs.zip)

https://github.com/user-attachments/assets/d1e02cb6-e541-45ba-83e5-10987d814519

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

**Comment by churik** [comment]
I don't think that it is feasible to finish investigation in 2.32, given the desired release cut date.
Deferring the issue, please be free to move it bak if you think there is a room for it @ilmotta

**Comment by ilmotta** [comment]
Based on the description in issue https://github.com/status-im/status-mobile/issues/21172

> Users experience issues with delayed or missing messages in community on both iOS and Android devices when reconnecting from offline to online (4G) .
> 
> -- https://github.com/status-im/status-mobile/issues/21172

Therefore, that issue is most likely related to this one, so I'm linking them.

@churik, makes sense to descope in 2.32, just as you descoped #21172

**Comment by ilmotta** [comment]
@churik, just one update on this one. Recently I aligned with @qfrank he would try to fix this issue and https://github.com/status-im/status-mobile/issues/21452. I think we could still try to fix them for 2.32, but if early next week we didn't manage it's fine to push for 2.33. @qfrank could you add assign yourself to these two issues if you are working on them?

**Comment by ilmotta** [comment]
@pavloburykh given that the issues below have been closed recently, could it be that this issue is also resolved?

- https://github.com/status-im/status-mobile/issues/21850
- https://github.com/status-im/status-mobile/issues/21452
- https://github.com/status-im/status-mobile/issues/21172

**Comment by pavloburykh** [comment]
> [@pavloburykh](https://github.com/pavloburykh) given that the issues below have been closed recently, could it be that this issue is also resolved?
> 
> * [Messages are delivered with delay #21850](https://github.com/status-im/status-mobile/issues/21850)
> * [Some messages are never received by the user who returned from offline #21452](https://github.com/status-im/status-mobile/issues/21452)
> * [Significant delay or missing messages on 4G in community  (without re-login or app closure) #21172](https://github.com/status-im/status-mobile/issues/21172)

Thank you @ilmotta. I have just re-checked in nightly and issue does not occur anymore. Closing the ticket.
